railties/test/isolation/abstract_unit.rb

```diff
@@ -311,10 +311,6 @@ def use_frameworks(arr)
     end
 
     def boot_rails
```

Comment:
this method wont do anything if this removal works.. I will do another pass to remove it, if we want to move forward with this.